.github/workflows/network-operator-docker-and-helm-ci.yaml

```diff
@@ -5,26 +5,31 @@ on:
     branches:
     - "master"
     - "v[0-9]+.[0-9]+.x"
+    tags:
+    - 'v[0-9]+.[0-9]+.[0-9]+*'
```

Comment:
nit: Probably better for code style consistency to use the same type of quotes throughout the file

Reply:
some parts it's syntactically important, for github actions. but not here - i'll change it, ty!